packages/cw-orch-mock/src/core.rs

```diff
@@ -310,6 +310,27 @@ impl<S: StateInterface> TxHandler for Mock<S> {
     }
 }
 
+impl WasmCodeQuerier for Mock {
+    /// Returns whether the checksum of the WASM file matches the checksum of the latest uploaded code for this contract.
+    fn get_contract_hash(&self, code_id: u64) -> Result<String, CwEnvError> {
+        let code_info = self.app.borrow().wrap().query_wasm_code_info(code_id)?;
```

Comment:
Does this actually return something? Is there some code in the `App` that handles this query? If so, what does it return? Can we add that information as a comment or doc-comment on this method?